modules/3d/include/opencv2/3d/odometry.hpp

```diff
@@ -0,0 +1,81 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+
+#ifndef ODOMETRY_HPP
```

Comment:
`OPENCV_3D_` prefix is mandatory.

@savuor Please re-read guidelines on Wiki to avoid missing of their requirements.

Reply:
done